src/com/znd/smash/commands/SmashChamps.java

```diff
@@ -56,4 +56,124 @@ private static ItemStack createPyroMenuKit() {
 		
 		return item;
 	}
+	
```

Comment:
You need to incorporate these item icons within the createChampMenu() function - above.

Reply:
done

Reply:
I don't see the updated code.

Reply:
Nevermind, I see it. My bad.